src/api/routes/mentorBookings.ts

```diff
@@ -30,6 +32,31 @@ const create = procedure
       topic,
     }, { transaction });
 
+    // If a specific mentor was requested, create a chat message
+    if (requestedMentorId) {
```

Comment:
remove this conditional. create a chat message regardless whether the mentor is specified.